src/com/content-event/event-list.js

```diff
@@ -236,7 +236,7 @@ export default class ContentEventList extends Component {
 			if ( user && user['id'] && lists && votes ) {
 				Body = page ? this.renderList(page) : null;
 			}
-			else if ( lists && !lists['1'] ) {
```

Comment:
this always checked if the **first** round has started—not the own we are currently viewing